src/screens/Feed/components/MediaModal/styles.js

```diff
@@ -0,0 +1,42 @@
+import { StyleSheet, Platform } from 'react-native';
+import * as colors from 'kitsu/constants/colors';
+import { scenePadding } from 'kitsu/screens/Feed/constants';
+
+export const styles = StyleSheet.create({
+  searchBox: {
+    marginHorizontal: 10,
+    marginBottom: 10,
+    marginTop: 2,
+    justifyContent: 'center',
+  },
+  searchBoxContainer: {
+    backgroundColor: colors.darkPurple,
+  },
+  rowPickerSeparator: {
+    height: StyleSheet.hairlineWidth,
+    backgroundColor: colors.lightGrey,
+  },
+  pickerRow: {
+    backgroundColor: '#FFFFFF',
+    padding: 8,
+  },
+  pickerIconCircle: {
+    width: 30,
+    height: 30,
+    borderRadius: 30,
```

Comment:
Is this correct? To achieve full circle border, radius should be `width / 2`

Reply:
Seems to work, i just took it from the feed select one